packages/destination-actions/src/destinations/linkedin-conversions/api/index.ts

```diff
@@ -14,4 +23,169 @@ export class LinkedInConversions {
       method: 'GET'
     })
   }
+
+  getAdAccounts = async (): Promise<DynamicFieldResponse> => {
+    try {
+      const response: Array<Accounts> = []
+      const result = await this.request<GetAdAccountsAPIResponse>(`${BASE_URL}/adAccountUsers`, {
+        method: 'GET',
+        searchParams: {
+          q: 'authenticatedUser'
+        }
+      })
+
+      result.data.elements.forEach((item) => {
+        response.push(item)
+      })
+
+      const choices = response?.map((item) => {
+        return {
+          label: item.role,
+          value: item.account
+        }
+      })
+
+      return {
+        choices: choices
```

Comment:
```suggestion
        choices
```
This should work yea?